projects/storefrontapp-e2e-cypress/cypress/sample-data/powertools-checkout-flow.ts

```diff
@@ -0,0 +1,7 @@
+import { SampleUser, user } from './checkout-flow';
+
+export const variantUser: SampleUser = {
+  ...user,
+  password: '12341234',
```

Comment:
Default password is not compliant with b2c complexity rules. Are we having 2 different sets of rules? One for b2c, another for b2c?